stable/nginx-ingress/README.md

```diff
@@ -64,6 +64,7 @@ Parameter | Description | Default
 `controller.service.externalIPs` | controller service external IP addresses | `[]`
 `controller.service.loadBalancerIP` | IP address to assign to load balancer (if supported) | `""`
 `controller.service.loadBalancerSourceRanges` | list of IP CIDRs allowed access to load balancer (if supported) | `[]`
+`controller.service.loadBalancerTerminateSSL` | terminates SSL at load balancer (forwards 443 to 80) | false
```

Comment:
We already have `controller.service.nodePorts.http(s)`. Thus, for consistency, you should use the following: `controller.service.targetPorts.http(s)`.